doc/convert-to-man

```diff
@@ -1,4 +1,5 @@
 #!/bin/sh
 
-pandoc -s -t man rg.1.md -o rg.1
-sed -i 's/\.TH.*/.TH "rg" "1"/g' rg.1
+pandoc -s -t man rg.1.md -o rg.1 &&
+sed -i.bak 's/\.TH.*/.TH "rg" "1"/g' rg.1 &&
+rm -f rg.1.bak # BSD `sed` requires the creation of a back-up file
```

Comment:
> I also added the && to make the script bail if any part of the process fails.

I usually put `set -e` at the top of scripts for that. Could we do that instead? It feels a little nicer than trailing every command with `&&`. :-)

(I'm not up on my shell compatibility, but if this means switching `#!/bin/sh` to `#!/bin/bash`, then I'm fine with that.)